### PR TITLE
Proofread ch02-00

### PR DESCRIPTION
**Pull request by joeljpresent**
I have proofread Chapter 02-00.

### CODE REVIEW

**Comment by Jimskapt** [comment]
Thank you so much for your work on this hudge file, Joël ! :tada:

I think the proofreading of next chapters will be shorter, now ! :wink: 

The ch03-00 and ch03-01 does not absolutely need proofreading because it was translated by @ShinySaana before I've merged and proofreaded them, but you can find them in `FR-ch03` branch, if you still want to proofreading it.

After `ch03-01`, I've split my translation for the other pages in separate branches (see #42, #43, and #46) as usual.

*(merging the existing translation of `FR-ch03` was a mistake, and i've not reproduced this with `ch04`)*

File: FRENCH/src/ch02-00-guessing-game-tutorial.md [hunk 1]
```diff
@@ -550,9 +555,9 @@ written this code as:
 -->
 
 Lorsque l'on appelle une méthode avec la syntaxe `.foo()`, il est généralement
-préférable d'ajouter une nouvelle ligne puis de l'indenter'indenter à l'aide
-d'espaces, afin de séparer les longues lignes de code. Nous aurions pu écrire ce
-code de cette manière :
+préférable d'ajouter un retour à la ligne puis d'indenter à l'aide d'espaces
+pour couper les longues lignes de code.
```
[the remark this answers is not in the document]
Comment (joeljpresent) [reply FRENCH/src/ch02-00-guessing-game-tutorial.md]: Yes.

File: FRENCH/src/ch02-00-guessing-game-tutorial.md [hunk 2]
```diff
@@ -1924,25 +1949,25 @@ $ cargo run
...
+Votre nombre : 59
+Vous avez gagné !
 Veuillez entrer un nombre.
-quit
-thread 'main' panicked at 'Please type a number!: ParseIntError { kind: InvalidDigit }', src/libcore/result.rs:785
+quitter
+thread 'main' panicked at 'Veuillez entrer un nombre !: ParseIntError { kind: InvalidDigit }', src/libcore/result.rs:785
 note: Run with `RUST_BACKTRACE=1` for a backtrace.
-error: Process didn't exit successfully: `target/debug/deduction` (exit code: 101)
+error: Process didn't exit successfully: `target/debug/supposition` (exit code: 101)
```
[the remark this answers is not in the document]
Comment (joeljpresent) [reply FRENCH/src/ch02-00-guessing-game-tutorial.md]: OK. This line doesn't even show up anyway.

File: FRENCH/src/ch02-00-guessing-game-tutorial.md [hunk 3]
```diff
@@ -1481,24 +1497,26 @@ that arm will execute and print `Too big!` to the screen. The `match`
...
-afficher à l'écran `C'est moins !`. L'expression `match` se termine ensuite, car
-il est inutile de consulter les autres branches de ce scénario.
+vérifier le motif de chaque branche.
+Elle consulte le motif de la première branche, `Ordering::Less` et remarque que
+la valeur `Ordering::Greater` ne correspond pas au motif `Ordering::Less` ;
+elle ignore donc le code de cette branche et passe à la suivante.
+Le motif de la branche suivante, `Ordering::Greater`, correspond à
+`Ordering::Greater` ! Le code associé à cette branche va être exécuté et va
+afficher à l'écran `C'est moins !`. L'expression `match` se termine ensuite, car
+elle n'a pas besoin de consulter les autres branches dans ce scénario.
```
[the remark this answers is not in the document]
Comment (joeljpresent) [reply FRENCH/src/ch02-00-guessing-game-tutorial.md]: OK for `de ce scénario`.

File: FRENCH/src/ch02-00-guessing-game-tutorial.md [hunk 4]
```diff
@@ -1265,14 +1278,15 @@ nombre entre 1 et 100.
...
+> Remarque : vous ne pourrez pas deviner quels *traits*, méthodes et
+> fonctions utiliser avec une *crate*. Les instructions pour l'utilisation
 > d'une *crate* se trouvent dans la documentation propre à chaque *crate*. Une
 > autre fonctionnalité intéressante de Cargo est que vous pouvez utiliser la
-> commande `cargo doc --open`, qui va construire la documentation intégrée dans
-> les dépendances et va l'ouvrir dans votre navigateur. Si par exemple vous vous
-> intéressez à d'autres fonctionnalités du *crate* `rand`, vous pouvez lancer
-> `cargo doc --open` et cliquer sur `rand` dans le volet latéral sur la gauche.
+> commande `cargo doc --open`, qui va construire localement la documentation
+> fournie par toutes vos dépendances et va l'ouvrir dans votre navigateur.
```
[the remark this answers is not in the document]
Comment (joeljpresent) [reply FRENCH/src/ch02-00-guessing-game-tutorial.md]: OK for `documentation intégrée`.

File: FRENCH/src/ch02-00-guessing-game-tutorial.md [hunk 5]
```diff
@@ -1027,12 +1039,12 @@ your code?
...
 Cargo embarque une fonctionnalité qui garantie que vous pouvez recompiler le
-même artéfact à chaque fois que vous ou quelqu'un d'autre compile votre code :
-Cargo va utiliser uniquement les mêmes versions de vos dépendances jusqu'à ce
-que vous demandiez le contraire. Par exemple, que se passe-t-il si la semaine
-prochaine version 0.3.15 du *crate* `rand` est publiée et qu'elle apporte une
-correction importante, mais aussi qu'elle produit une régression qui va casser
-votre code ?
+même artéfact à chaque fois que vous ou quelqu'un d'autre compile votre code :
+Cargo va utiliser uniquement les versions de dépendances que vous avez
+spécifiées jusqu'à ce que vous indiquiez le contraire.
```
[the remark this answers is not in the document]
Comment (joeljpresent) [reply FRENCH/src/ch02-00-guessing-game-tutorial.md]: OK for _que vous avez utilisé**es**_.

File: FRENCH/src/ch02-00-guessing-game-tutorial.md [hunk 6]
```diff
@@ -632,27 +639,29 @@ can use it. In this case, that value is the number of bytes in what the user
...
 argument de `expect`. Si l'appel à `read_line` retourne une variante `Err`, ce
-sera probablement dû à une erreur d'un élément du système d'exploitation. Si par
-contre `read_line` a pour valeur la variante `Ok`, `expect` récupèrera le
-contenu du `Ok`, qui est le résultat de l'operation, et vous le retournera afin
+sera probablement dû à une erreur du système d'exploitation. Si en revanche
+`read_line` a pour valeur la variante `Ok`, `expect` récupèrera le
+contenu du `Ok`, qui est le résultat de l'opération, et vous le retournera afin
 que vous puissiez l'utiliser. Dans notre exemple, ce résultat est le nombre
-d'octets que l'utilisateur a saisi dans l'entrée standard.
+d'octets que l'utilisateur a saisis dans l'entrée standard.
```
[the remark this answers is not in the document]
Comment (joeljpresent) [reply FRENCH/src/ch02-00-guessing-game-tutorial.md]: It's a plural form which agrees with the direct object complement.
But it depends on whether we consider that the direct object is `octets` (`L'utilisateur a saisi des octets`) or `nombre d'octets` (`L'utilisateur a saisi un [certain] nombre d'octets`).
In the latter case, you can revert it back to `a saisi`.

File: FRENCH/src/ch02-00-guessing-game-tutorial.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Jimskapt) [reply FRENCH/src/ch02-00-guessing-game-tutorial.md]: Nope, I was wrong, i was thinking it prints the project name, but finally it was the name of the variable.

So finally, we will effectly keep `target/debug/supposition` :sweat_smile:

**Pull request merged**